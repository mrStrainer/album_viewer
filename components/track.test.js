import React from 'react';
import Track from './track';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<Track />).toJSON();
  expect(rendered).toMatchSnapshot();
});
