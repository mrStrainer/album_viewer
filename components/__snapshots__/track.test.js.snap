// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "#fff",
      "padding": 2,
    }
  }
>
   
   
</Text>
`;
